vllm/model_executor/layers/quantized_linear/smoothquant.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+import torch
+from vllm.model_executor.parallel_utils.tensor_parallel.layers import (
+    ColumnParallelLinear, RowParallelLinear)
+from vllm.i8cugemm import I8CUGEMM
+
+i8cugemm = I8CUGEMM()
+
+
+class SQColumnParallelLinear(ColumnParallelLinear):
+
+    def create_weights(self, dtype: torch.dtype) -> None:
+        assert self.input_size % self.quant_config.weight_bits == 0
+        assert dtype in [torch.half, torch.float16, torch.float]
+        self.register_buffer(
+            'weight',
+            torch.randint(-127,
```

Comment:
any particular reason to use `torch.randint`? `torch.empty` is much faster

Reply:
No, we can switch to torch.empty. This might be the reason for the slow loading. We haven't tested it with the 70B model yet. We were focusing on the 13B model, but we will try the larger model and address the slow loading problem.

Reply:
i tried torch.empty and it's still working and much faster in loading